docs/buildtest_perlmutter.rst

```diff
@@ -0,0 +1,116 @@
+Buildtest Tutorial on Perlmutter
+===================================
+
+This tutorial will be conducted on `Perlmutter <https://docs.nersc.gov/systems/perlmutter/>`_ system. If you need account access please
+`obtain a user account <https://docs.nersc.gov/accounts/>`_.
+
+Setup
+------
+
+Once you have a NERSC account, you can `connect to NERSC system <https://docs.nersc.gov/connect/>`_. You will need access to a
+terminal client and ssh into perlmutter as follows::
```

Comment:
Suggestion: 'You will need access to ssh. With that you can connect to perlmutter as follows'